test/lib/build/ProjectBuilder.js

```diff
@@ -293,9 +297,9 @@ test.serial("build: Multiple projects", async (t) => {
 		"project.b", "project.c"
 	], "_createRequiredBuildContexts got called with correct arguments");
 
-	t.is(requiresBuildAStub.callCount, 2, "TaskRunner#requiresBuild got called twice times for library.a");
-	t.is(requiresBuildBStub.callCount, 2, "TaskRunner#requiresBuild got called twice times for library.b");
-	t.is(requiresBuildCStub.callCount, 2, "TaskRunner#requiresBuild got called twice times for library.c");
+	t.is(requiresBuildAStub.callCount, 1, "TaskRunner#requiresBuild got called twice times for library.a");
```

Comment:
TaskRunner#requiresBuild got called once for

Reply:
Thanks! Done 👍